src/main/java/mods/eln/sixnode/lampsocket/LightBlockEntity.java

```diff
@@ -135,29 +135,30 @@ public void updateEntity() {
             }
         }
 
-        if (light != worldObj.getBlockMetadata(xCoord, yCoord, zCoord)) {
-
-            worldObj.setBlockMetadataWithNotify(xCoord, yCoord, zCoord, light, 2);
-            worldObj.updateLightByType(EnumSkyBlock.Block, xCoord, yCoord, zCoord);
+        if (light != worldObj.getBlockState(this.pos).getBlock().getMetaFromState(worldObj.getBlockState(this.pos))) {
+            worldObj.getBlockState(this.pos).getBlock().setLightLevel(2);
+            //TO BE FIXED
```

Comment:
Use the syntax: "// TODO: <whatever>".

This is so it's easy to search for them later.

Reply:
Also a _de facto_ standard--most IDEs worth their salt collect "TODO lists" because that particular word in that usage is prevalent enough :)